common/wonders/wc_wonders_historical.txt

```diff
@@ -247,3 +247,395 @@ wonder_wyrmrest_temple = {
 	stage = {
 	}
 }
+wonder_ahnqiraj = {
```

Comment:
It's a thing from https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1307. Should be reverted or this branch should be based on  https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1307.

Reply:
It's not resolved!